assignments/arrays.js

```diff
@@ -70,27 +70,55 @@ console.log(`Car 33 is a *car year goes here* *car make goes here* *car model go
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+let inventoryLength = inventory.length;
+console.log()
+console.log(inventoryLength);
+console.log(("Make: " +inventory[ inventoryLength-1].car_make + " Model: " +inventory[ inventoryLength-1].car_model));
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+let inventoryLength1 = inventory.length;
+
+for (i=0; i< inventoryLength1;i++){
+
+ carModels[i] = inventory[i].car_model
```

Comment:
Line 86: This works, however I would recommend using `carModels.push()` to add the elements. This way is fine for this problem because you're doing it for every element in `inventory`, but it will leave holes in an array (the element at some indexes would be `undefined`) if you were to only add some elements.